<?xml version="1.0" encoding="UTF-8"?>
<testrun name="SolrTests" project="SolrTestApp1" tests="7" started="7" failures="2" errors="1" ignored="0">
  <testsuite name="SolrTests" time="0.271">
    <testcase name="test_basic" classname="SolrTests" time="0.0"/>
    <testcase name="test_GetDocFromSolrService_UnitTest" classname="SolrTests" time="0.201"/>
    <testcase name="test_GetDocFromSolrService_AcceptanceTest" classname="SolrTests" time="0.01"/>
    <testcase name="test_GetDocFromSolrService_mapsName_AcceptanceTest" classname="SolrTests" time="0.02"/>
    <testcase name="test_GetDocFromSolrService_mapsSortName_AcceptanceTest" classname="SolrTests" time="0.02">
      <failure>
        <expected>pink floyd</expected>
        <actual>Pink Floyd</actual>org.junit.ComparisonFailure: expected:&lt;[pink f]loyd&gt; but was:&lt;[Pink F]loyd&gt;&#13;
	at org.junit.Assert.assertEquals(Assert.java:125)&#13;
	at org.junit.Assert.assertEquals(Assert.java:147)&#13;
	at SolrTests.test_GetDocFromSolrService_mapsSortName_AcceptanceTest(SolrTests.java:41)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:168)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:134)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:110)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:128)&#13;
	at junit.framework.TestResult.run(TestResult.java:113)&#13;
	at junit.framework.TestCase.run(TestCase.java:124)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:243)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:238)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="test_GetDocFromSolrService_mapsUrl_AcceptanceTest" classname="SolrTests" time="0.01">
      <failure>java.lang.AssertionError: expected:&lt;pink-floyd&gt; but was:&lt;null&gt;&#13;
	at org.junit.Assert.fail(Assert.java:93)&#13;
	at org.junit.Assert.failNotEquals(Assert.java:647)&#13;
	at org.junit.Assert.assertEquals(Assert.java:128)&#13;
	at org.junit.Assert.assertEquals(Assert.java:147)&#13;
	at SolrTests.test_GetDocFromSolrService_mapsUrl_AcceptanceTest(SolrTests.java:49)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:168)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:134)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:110)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:128)&#13;
	at junit.framework.TestResult.run(TestResult.java:113)&#13;
	at junit.framework.TestCase.run(TestCase.java:124)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:243)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:238)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="test_GetTop10DocumentsFromSolrService_AcceptanceTest" classname="SolrTests" time="0.0">
      <error>java.lang.NullPointerException&#13;
	at SolrTests.test_GetTop10DocumentsFromSolrService_AcceptanceTest(SolrTests.java:57)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:168)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:134)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:110)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:128)&#13;
	at junit.framework.TestResult.run(TestResult.java:113)&#13;
	at junit.framework.TestCase.run(TestCase.java:124)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:243)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:238)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
